lib/testLauncher/SuitestLauncher.js

```diff
@@ -52,6 +52,7 @@ class SuitestLauncher {
 						shell: true,
 						env: {
 							...process.env,
+							[envVars.SUITEST_LAUNCHER_PROCESS]: true,
```

Comment:
I wonder if `true` is a good value. Would it be converted to string upon transfer to Chile process? If so, maybe it's a good idea to use here some string value to avoid confusion.